### PR TITLE
ADEN-3483 Sync UAP on Mercury

### PR DESCRIPTION
**Pull request by drets**
App part for https://github.com/Wikia/mercury/pull/2591

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/gpt/helper.js [hunk 1]
```diff
@@ -76,9 +76,8 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargeting.rv = count.toString();
 			}
 		}
-		if (uapId) {
-			slotTargeting.uap = uapId.toString();
-		}
+
+		slotTargeting.uap = uapId ? uapId.toString() : 'none';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js:80]: Why do we need this `none`?

Edit: OK, I think I know why - to reset value on consecutive pageviews (?). But slotTargeting is copied (here: https://github.com/Wikia/app/pull/10790/files#diff-d0456a56fa8978685b777a3c10aaf5abR65) so is this resetting needed?

Comment (drets) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js:80]: Yes and yes.
(for some reasons gpt lib remembers what was set before and in order to update slot targeting parameter (in requested URL) we need to set a new not-falsy value)
Do you know a better way how to reset `uap` param on consecutive pageviews on Mercury?

Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js:80]: Yep, @fraszczakszymon you're our last resort. We were looking for a way to "reset" slot targeting with pure GPT but it seems there is no way. And resetting it by setting it to falsy values results in our wrapper and implementation working correctly but the request to DFP includes the old `uap` on consecutive PV in Mercury :/

**Comment by nandy-andy** [comment]
LGTM :+1: Let's test it.

**Comment by nandy-andy** [comment]
LGTM

**Pull request merged**